clusterloader2/pkg/prometheus/manifests/prometheus-rules.yaml

```diff
@@ -63,6 +63,15 @@ spec:
       record: probes:dns_lookup_latency:histogram_quantile
       labels:
         quantile: "0.50"
+    - expr: |
+        probes_dns_propagation_count
+      record: probes:dns_propagation_count
+    - expr: |
+        probes_dns_propagation_seconds
+      record: probes:dns_propagation_seconds
+    - expr: |
+        sum(max_over_time(probes_dns_propagation_seconds[1d]))/sum(max_over_time(probes_dns_propagation_count[1d]))
```

Comment:
Why do we have hardcoded 1d here?

Reply:
As the timestamp itself isn't relevant here, we need to aggregate the measurements in order to calculate the average. As Prometheus measurements are time series by nature, the idea is to make a query that would show the last value as the true value, so when shown on Grafana with a gauge chart, it will display the correct number. As such, we need to extend the time to cover the entire timespan of the experiment (i.e., from measurement start to measurement gather). 1d seemed sufficient; we can add a very high number (e.g., 10000d) to be sure to cover all possible experiments.

Please note that this record is only used in the context of Grafana; the one used in CL2 where the percentile values are calculated is probes:dns_propagation_seconds.

Reply:
Thx for the rationale.

I believe we should be good with just `1h`, though - I wouldn't expect to measure the propagation latency for anything longer than 10 minutes, and, even if, `1h` should be more than enough.

Reply:
Fixed